.typos.toml

```diff
@@ -0,0 +1,4 @@
+[files]
+extend-exclude = ["components/dada-web/ace/", "book/src/css/speech-bubbles.css", "book/package-lock.json"]
+[default.extend-words]
+fo = "fo"
```

Comment:
What is this for?

Reply:
`extend-exclude` excludes the listed files from consideration and will therefore not be checked for spelling mistakes. I.e., allow typos in third party files + package-lock.json.
`extend-words` allows the listed words and will not treat them as a typo. This was added to allow the examples here:
https://github.com/dada-lang/dada/blob/main/book/docs/reference/raw-string-literals.md